source/index.html

```diff
@@ -1,39 +1,282 @@
+                        <li>1/2 hour walk on beach/or woods. Car ride to and from designated area with 1/2 hour of play time $12.00</li>
+                        <li>Filled frozen kong treat $5</li>
+                        <li>Daily brushing $5</li>
+                        <li>10% off grooming</li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        <footer>
+            <section>
+                <img class="dog-divider" src="./assets/new-dog6.png">
+                <hr>
+                <div class="content">
+                    <h1>POETS PET PARLOR</h1>
+                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2743.1428642649557!2d-87.41570234910645!3d46.56464007902694!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4d51df4d52a6b0cd%3A0xfc299af6713a58b7!2sPoets+Pet+Parlor!5e0!3m2!1sen!2sus!4v1474386192906" width="auto" height="230" frameborder="0" style="border:0" allowfullscreen></iframe>
+                    <table class="foot-nav">
+                        <thead>
+                            <td>Navigation</td>
+                        </thead>
+                        <tr>
+                            <td>
+                                <hr>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td><a href="#grooming">Grooming Services</a></td>
+                        </tr>
+                        <tr>
+                            <td><a href="#daycare">Daycare</a></td>
+                        </tr>
+                        <tr>
+                            <td><a href="#boarding">Boarding</a></td>
+                        </tr>
+                        <tr>
+                            <td><a href="./assets/daycare_form.pdf" download="daycare form">Enrollment Form</a></td>
+                        </tr>
+                    </table>
+                    <table class="foot-nav">
+                        <thead>
+                            <td>Socialize with us</td>
+                        </thead>
+                        <tr>
+                            <td>
+                                <hr>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td><a href="https://www.facebook.com/poetspetparlor/?fref=ts">Facebook</a></td>
+                        </tr>
+                        <tr>
+                            <td><a href="https://www.instagram.com/poets_pet_parlor/">Instagram</a></td>
+                        </tr>
+                        <!-- <tr>
+                            <td>Twitter</td>
+                        </tr>
+                        <tr>
+                            <td>Snapchat</td>
+                        </tr>-->
+                    </table>
+                    <div class="copyright">
+                        <p><img src="./assets/copyright-32.png">. Pot Pie Digital. 2016.</p>
```

Comment:
I think instead of the `./assets/copyright-32.png` you could just do `&copy;` … doesn't need a period between it and "Pot" either.